features/step_definitions/litigator_transfer_claim_steps.rb

```diff
@@ -23,6 +23,11 @@
   @litigator_transfer_claim_form_page.transfer_detail.transfer_date.set_date date_string
 end
 
+And(/^I enter the transfer date (.*) years ago$/) do |years|
```

Comment:
To get `years` as an integer the step can be defined without a regular expression:

```suggestion
And('I enter the transfer date {int} years ago') do |years|
```

The documentation is [here.](https://cucumber.io/docs/cucumber/step-definitions/?lang=ruby)

You can then create a date as `years.years.ago` instead of relative to `Time.zone.today`.